packages/core/src/storybook/decorators/withLiveEdit/withLiveEdit.module.scss

```diff
@@ -6,4 +6,5 @@
   right: 0;
   border-top-left-radius: 4px;
   overflow: hidden;
+  z-index: 1;
```

Comment:
If we don't insert the LiveEditor changes for actionComp, this also might be redundant, as the actions would be after the cm-editor in the DOM